zips/zip-0271.md

```diff
@@ -3,7 +3,7 @@
     Owners: Daira-Emma Hopwood <daira@jacaranda.org>
             Kris Nuttycombe <kris@nutty.land>
             Jack Grigg <jack@electriccoin.co>
-    Status: Draft
+    Status: Proposed
```

Comment:
Process failure: per ZIP 0:
> A ZIP SHOULD only change status from Draft (or Rejected) to Proposed, when the Owner deems it is complete and there is rough consensus on the forums, validated by consensus among the current ZIP Editors. **If it's a Consensus ZIP, a Deployment section MUST be present in order for the ZIP to change status to Proposed.** Typically, although not necessarily, this will specify a network upgrade in which the consensus change is to activate.

ZIP 271 does not contain a Deployment section.